tests/Tests/Unit/FHIR/SMART/SMARTLaunchTokenTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
header

btw, the OpenEMR Fans love the testing!
![2DV](https://user-images.githubusercontent.com/278968/102939860-ec30bc00-4463-11eb-8a05-537882dfeb8d.gif)

Reply:
now if only we could get them to sponsor some projects:)

Reply:
I'll take the moon for payment as long as they can figure out the whole shrinking for longer periods of time thing.